magpie/api/exception.py

```diff
@@ -407,29 +403,29 @@ def generate_response_http_format(httpClass, httpKWArgs, jsonContent, outputType
     jsonContent = str(jsonContent) if not isinstance(jsonContent, six.string_types) else jsonContent
 
     # adjust additional keyword arguments and try building the http response class with them
-    httpKWArgs = dict() if httpKWArgs is None else httpKWArgs
+    http_kwargs = dict() if http_kwargs is None else http_kwargs
     try:
         # directly output json
         if outputType == CONTENT_TYPE_JSON:
             json_type = "{}; charset=UTF-8".format(CONTENT_TYPE_JSON)
-            httpResponse = httpClass(body=jsonContent, content_type=json_type, **httpKWArgs)
+            httpResponse = httpClass(body=jsonContent, content_type=json_type, **http_kwargs)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "httpResponse" doesn't conform to snake_case naming style](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)